app/presenters/tree_builder_report_saved_reports.rb

```diff
@@ -36,7 +36,10 @@ def x_get_tree_roots(count_only, _options)
   end
 
   def x_get_tree_custom_kids(object, count_only, _options)
-    scope = MiqReportResult.with_current_user_groups_and_report(from_cid(object[:id].split('-').last))
+    report_id = from_cid(object[:id].split('-').last)
+    scope = MiqReportResult.with_current_user_groups_and_report(report_id)
+                           .select('miq_report_results.*', 'miq_tasks.status AS task_status')
+                           .joins(:miq_task)
```

Comment:
Can this be done at the model level without resorting to .select and .joins in the presenter?  Mostly just curious...I instantly react when I see things like the select fragments.